ISHPermissionKit/Requests/ISHPermissionRequestNotificationsRemote.m

```diff
@@ -0,0 +1,26 @@
+//
+//  ISHPermissionRequestNotificationsRemote.m
+//  ISHPermissionKit
+//
+//  Created by Felix Lamouroux on 01.09.2015.
+//  Copyright (c) 2014 iosphere GmbH. All rights reserved.
+//
+
+#import "ISHPermissionRequestNotificationsRemote.h"
+#import "ISHPermissionRequest+Private.h"
+
+@interface ISHPermissionRequestNotificationsLocal ()
+@property (copy) ISHPermissionRequestCompletionBlock completionBlockRemote;
+@end
+
+@implementation ISHPermissionRequestNotificationsRemote
+
+- (ISHPermissionState)permissionState {
+    if (![[UIApplication sharedApplication] respondsToSelector:@selector(isRegisteredForRemoteNotifications)]) {
+        return ISHPermissionStateUnsupported;
+    }
+    
+    return [super permissionState];
```

Comment:
Should we not return `[[UIApplication sharedApplication] isRegisteredForRemoteNotifications]`?